### PR TITLE
Update codebridge colors/styling

### PR DESCRIPTION
**Pull request by molly-moen**
This PR is mostly font/background color updates with some margins and borders sprinkled in. The changes here do minorly impact Music Lab and Java Lab, see the screenshots for details. 

The goal here was to match the [figma](https://www.figma.com/design/vcNGOFsIks0HhRk6GEOfZf/Upper-Lab?node-id=4439-92405&node-type=frame&m=dev) as closely as possible. Most of this was slightly changing shades of gray. This impacts Music Lab because we are changing the panel container header color. We discussed this on [slack](https://codedotorg.slack.com/archives/C06JELCAPT9/p1727303549960629) and in meetings and decided to go with the new color. I also tried to use the design system constants as much as possible: changes from `neutral_dark` to `light_gray` with the same number modifier at the end are the same color.

I also updated the codemirror dark mode background color, which impacts Java Lab. It's a very minimal change. When doing so I noticed the constant and line number color did not have high enough contrast with the background color, so I upped the contrast. [Slack thread](https://codedotorg.slack.com/archives/C06JELCAPT9/p1727815116509039) discussing that.

#### Before
Screenshot 2024-10-01 at 3 49 54 PM

#### After
Screenshot 2024-10-01 at 3 50 41 PM

### Music Lab
Only the header color changed in Music Lab.
#### Before
Screenshot 2024-10-01 at 3 51 02 PM

#### After
Screenshot 2024-10-01 at 3 50 49 PM

### Java Lab
Only the background color of the editor and font color of comments/line numbers changed.
#### Before
Screenshot 2024-10-01 at 3 47 17 PM

#### After
Screenshot 2024-10-01 at 3 50 54 PM

## Links

- jira ticket: [CT-807](https://codedotorg.atlassian.net/browse/CT-807)
- [figma](https://www.figma.com/design/vcNGOFsIks0HhRk6GEOfZf/Upper-Lab?node-id=4439-92405&node-type=frame&m=dev)

## Testing story
Tested locally. All changes are cosmetic.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/codebridge/Editor/styles/editor.module.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@
 .editorContainer {
   width: 100%;
   height: 100%;
-  background-color: #292f36;
+  background-color: $light_black;
```
Comment (molly-moen) [inline apps/src/codebridge/Editor/styles/editor.module.scss:6]: these are actually the same color, we just weren't using the constant before.

**Review by cnbrenci — APPROVED** [review]
lgtm! 🚀

File: apps/src/codebridge/Workspace/WorkspaceAndConsole.tsx [hunk 2]
```diff
@@ -80,7 +81,7 @@ const WorkspaceAndConsole: React.FunctionComponent = () => {
         resizeItemTop={() => PANEL_TOP_COORDINATE}
         position={editorHeight}
         onResize={handleResize}
-        style={{position: 'static'}}
+        style={{position: 'static', backgroundColor: color.light_gray_950}}
```
Comment (thomasoniii) [inline apps/src/codebridge/Workspace/WorkspaceAndConsole.tsx:84]: should this be pulled out into its own class?

**Review by thomasoniii — APPROVED** [review]
minor style/class question, but otherwise lgtm 🚀

File: apps/src/codebridge/Workspace/WorkspaceAndConsole.tsx (hunk 2 above)
Comment (molly-moen) [reply apps/src/codebridge/Workspace/WorkspaceAndConsole.tsx:84]: `HeightResizer` is an old component that doesn't have the option to pass in a class name, and it uses styles that we are partially overriding here (position). I'm going to leave as-is for now, that component could use a refactor away from radium 😄

**Pull request merged**